### PR TITLE
Fix for v0.5

### LINKED ISSUE #6: No longer works on Julia 0.5
On Julia 0.5- master, the following [parametric definition for `Base.call`](https://github.com/pjabardo/CurveFit.jl/blob/0d8900f6af14e2897001de7119c089be76dea419/src/linfit.jl#L121-L123
) is no longer allowed:

```jl
import Base.call

call{T<:LeastSquares}(f::T, x) = apply_fit(f, x)
```

ref: JuliaLang/julia#14919

### PR DESCRIPTION
**Pull request by ranjanan**
Fixes #6

### CODE REVIEW

File: REQUIRE [hunk 1]
```diff
@@ -1,2 +1,3 @@
 julia 0.4
-Polynomials
\ No newline at end of file
+Polynomials
+Compat
```
Comment (ararslan) [inline REQUIRE]: Should probably add a minimum version for Compat, e.g. `Compat 0.8.4`.

Comment (ranjanan) [inline REQUIRE]: Ah yes, my bad.

**Comment by ararslan** [comment]
Looks good to me!

**Comment by tkelman** [comment]
@pjabardo it would be a very good idea to enable Travis CI here. We can also easily modernize the .yml file to use `language: julia` support.

**Comment by pjabardo** [comment]
Sorry for not answering before, I was out. Tonight I will do this
 

    Em Terça-feira, 19 de Julho de 2016 3:27, Tony Kelman <notifications@github.com> escreveu:
 

 @pjabardo it would be a very good idea to enable Travis CI here. We can also easily modernize the .yml file to use language: julia support.—

**Pull request merged**